lua/lspconfig/ui/lspinfo.lua

```diff
@@ -264,12 +264,22 @@ return function()
     end
   end
 
+  local available_servers = util.available_servers()
   local matching_config_header = {
     '',
-    'Configured servers list: ' .. table.concat(util.available_servers(), ', '),
+    'Configured servers list: ' .. table.concat(available_servers, ', '),
   }
-
+  local deprecate_servers = {}
   vim.list_extend(buf_lines, matching_config_header)
+  local aliased = require('lspconfig').server_alias
+  for server_name, deprecate in pairs(aliased) do
+    if deprecate.inconfig then
```

Comment:
doesn't lspinfo.lua already show the "current connected" servers? why do we need to store state in `inconfig`, instead of just using `server_alias()` here just-in-time to see if a currently-connected server is deprecated?

Reply:
because available just show then server which list in configs. after deprecate there will show the new server name . like user configed `tsserver` but when they call LspInfo  available servers show `ts_ls` already.